docs/topics/db/aggregation.txt

```diff
@@ -632,3 +632,46 @@ aggregate that author count, referencing the annotation field:
     >>> from django.db.models import Avg, Count
     >>> Book.objects.annotate(num_authors=Count("authors")).aggregate(Avg("num_authors"))
     {'num_authors__avg': 1.66}
+
+Handling empty querysets using aggregate ``default`` 
+----------------------------------------------------
+
+.. versionadded:: 4.0
+
+Aggregation functions return ``None`` when used with an empty ``QuerySet``.
```

Comment:
Not just empty querysets but empty groups when grouping 👍  (see my comments in the ticket)

Not just for this sentence  but the paragraph need to keep this in mind.

Reply:
Thanks a lot for the comments friends. I am working on the necessary adjustments.